### PR TITLE
add Addition of other Iranian domains

### PR DESCRIPTION
**Pull request by alirezax5**
I made some changes on Iranian domains for better classification:
Added :

-category-ir-appsandgame
-category-ir-soical
-category-ir-porn
-category-ir-vod
-category-ir-ispAndDatacenter
-category-ir-game
Update :

-category-ir
-category-ir-media
-category-porn

### CODE REVIEW

**Comment by rootmelo92118** [comment]
Do not do the same operation to PR the next time. It looks like you deleted all contents in one file and submit the new data including the original contents. It will make it difficult for maintainers to read your submissions. Just submit the new addition or modification you want.

**Comment by rootmelo92118** [comment]
And sort your data after your modifications.

**Comment by rootmelo92118** [comment]
And this PR has conflict in `category-porn`. Rebase it.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]
> And this PR has conflict in `category-porn`. Rebase it.

@alirezax5 there are 4,632 additions & exact 4,632 deletions on the `data/category-porn` file, which has nothing to do with this PR.
I wonder whether it is a mistake during the contribution, please double-check your commit log.

File: data/category-porn [hunk 1]
```diff
@@ -1,4632 +1,4632 @@
...
-regexp:(^|\.)zzz[1-9][0-9]\.top$
+include:18comic
+include:54647
+include:avmoo
+include:bilibili2
+include:bongacams
+include:boylove
+include:brazzers
+include:bttzyw
+include:category-ir-porn
```
Comment (rootmelo92118) [inline data/category-porn:9]: @IceCodeNew Well. He just only add one line in here.

**Comment by rootmelo92118** [comment]
go to #1394

**Comment by alirezax5** [comment]
I'm sorry for the problems, I'm trying to add the important domains properly after the changes in #1644 .

I also noticed that some porn domains of Iranian websites have been added to the porn category
There is no need to create a category for Iranian porn sites?

And one point that I saw, because snapp.ir is a super app, it has several other domains and they are divided into categories such as shopping.
Isn't it better to add them all inside a file called snapp?

**Comment by rootmelo92118** [comment]
> I'm sorry for the problems, I'm trying to add the important domains properly after the changes in #1644 .
> 
> I also noticed that some porn domains of Iranian websites have been added to the porn category There is no need to create a category for Iranian porn sites?
> 
> And one point that I saw, because snapp.ir is a super app, it has several other domains and they are divided into categories such as shopping. Isn't it better to add them all inside a file called snapp?

I see.

**Pull request closed without merging**